### ISSUE TITLE
Yukawa signs in decoupling model

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#455]
The decoupling scheme sets BSM Yukawas as
```cpp
upQuarksDRbar.diagonal()    = sm.get_MFu();
...
MODEL->set_Yu((((1.4142135623730951*upQuarksDRbar)/v).transpose()).real());
```
This makes Yukawas always positive. Is it possible that originally a model had negative Yukawa? Would it make sense to write
```cpp
MODEL->set_Yu(
   std::copysign(
      (((1.4142135623730951*upQuarksDRbar)/v).transpose()).real(),
      MODEL->get_Yu()
   );
```
(maybe `get_Yu` should have some indices).

**Comment by Expander** [comment]
Good point! Do I understand correctly that `MODEL->get_Yu().cwiseSign()` is a matrix with only either `1` or `-1` elements, which represent the sign of the elements in `MODEL->get_Yu()`?

**Comment by wkotlarski** [comment]
Yes. According to documentation, `cwiseSign` is `an expression of the coefficient-wise signum of *this.`

**Comment by pathron** [comment]
Thanks for spotting this.   There are models with a minus sign in the Yukawas right?  So this would be wrong in those cases?

**Comment by wkotlarski** [comment]
I don't know if we currently have such a model. But nothing stops a user from writing one. It thought about it after I've noticed that in the build in SM yu < 0. So such models are possible.
